lib/context.js

```diff
@@ -197,6 +197,14 @@ class ContextSession {
     debug('create session with val: %j externalKey: %s', val, externalKey);
     if (this.store) this.externalKey = externalKey || this.opts.genid();
     this.session = new Session(this.ctx, val);
+    if (!this.opts.autoCommit) {
```

Comment:
I'd like to move this method here https://github.com/koajs/session/blob/master/lib/session.js . and if we named the method as `manuallyCommit`,I think it's ok to always add this method even `autodCommit` was set to true.

Reply:
@dead-horse `Session` has no reference to ContextSession so I had to pass it.

Reply:
@dead-horse Updated!